libcxx/modules/std/ranges.inc

```diff
@@ -274,6 +274,15 @@ export namespace std {
     } // namespace views
 #endif // _LIBCPP_STD_VER >= 23
 
+#if _LIBCPP_STD_VER >= 23
+    // [range.stride], stride view
+    using std::ranges::stride_view;
+
+    namespace views {
+      using std::ranges::views::stride;
+    }  // namespace views
+#endif // _LIBCPP_STD_VER >= 23
+
```

Comment:
Why did you move this block? The order of the declaration is the order of declarations in the header's synopsis.

Reply:
I did not mean to move it. I think that is an artifact from how `ranges.inc` has changed over time as I have worked on this patch. I am terribly sorry for screwing up. I will fix it.

Reply:
no worry, I just wondered why this was changed.

Reply:
I think that I got this in the most recent fixup. (700d38e) Let me know if I managed to get it back in the proper spot!